### ISSUE TITLE
Standard keyboard shortcuts and navigation not working

### ISSUE THREAD

**Issue by NiklasBr** [gifox/tracker#45]
There are a few places in the app where keyboard navigation should be possible but is not. A couple of examples:

 * `⌘W` can't be used to close the settings window.
 * `Up` and `Down` arrow keys can't be used to make changes to numeric fields such as Record fps and Playback fps.
 * `Left` and `Right` arrow keys can't be used to move the input cursor in the record modal window.
 * `Tab` key can't be used to move focus between input fields in the record modal window.

Minor issues, true, but things which would be awesome if fixed!

**Comment by ianbytchek** [comment]
Yep, this is a long standing issue and annoying as hell, copy / paste is probably the worst of it. This will be fixed in the near future. Apologies for this.

**Comment by ianbytchek** [comment]
@NiklasBr thanks for awesome and detailed suggestion. Most available in the next release! 🎉
